packages/starlight/virtual.d.ts

```diff
@@ -7,3 +7,16 @@ declare module 'virtual:starlight/plugin-translations' {
 	const PluginTranslations: import('./utils/plugins').PluginTranslations;
 	export default PluginTranslations;
 }
+
+declare module 'virtual:starlight/project-context' {
+	const ProjectContext: {
+		root: string;
+		srcDir: string;
+		trailingSlash: import('astro').AstroConfig['trailingSlash'];
+		build: {
+			format: import('astro').AstroConfig['build']['format'];
+		};
+		legacyCollections: boolean;
+	};
+	export default ProjectContext;
+}
```

Comment:
This move from `virtual-internal.d.ts` → `virtual.d.ts` is required due to the new import of `virtual:starlight/project-context` in `packages/starlight/utils/translations.ts` which would now fail for users [running `tsc --noEmit`](https://github.com/withastro/starlight/blob/main/docs/package.json#L12) on their project.